js/script.js

```diff
@@ -27,11 +27,23 @@ let currentFilter = ALL;
 let newTaskBeingEdited = false;
 let newTaskId = null;
 
+const loadTasksFromLocalStorage = () => {
+  const storedTasks = localStorage.getItem("tasks");
```

Comment:
```suggestion
  const storedTasks = localStorage.getItem(TASKS);
```

Import it from `js/helpers/constants.js`